general/support-team.md

```diff
@@ -1,19 +1,19 @@
 [<- back to index](../README.md)
 
 # About the Support Team 
-## Getting Added to the Support Team 
+## Joining the Support Team 
 1. Current support team agrees that there is a need to add a new support team member. Reasons for this include: 
   - Workload is heavy, and more team members would help reduce workload for everyone **(this is the most common reason someone is added to the support team)** 
   - New perspective on a specific issue that affects Django Girls 
   - New perspective on the general direction of Django Girls 
-2. A support team member nominates a new person. A person is usually nominated on the following basis: 
+  - Terms of past members expired, or they've decided to leave the Support Team
+2. Support Team announces a nomination period for new members, and allow all community members to nominate someone who they think would be a good addition to the team. Nominations are opened for two weeks, and are public in form of Pull Requests to Django Girls repository. Self-nominations are more than welcome! A person is usually nominated on the following basis: 
```

Comment:
I'm not 100% sure if the PR are the best way to handle nominations, since I'm worried about personal data being exposed, especially when you nominate someone else.